lib/helium/collection.rb

```diff
@@ -80,6 +80,36 @@ def last
       collection.last
     end
 
+    # Adds resources of the same type to the collection related to the
+    # '@belongs_to' resource
+    def add_relationships(items)
+      body = relationship_request_body(items)
+
+      @client.post(relationship_path, body: body) if relationship_path
+
+      self
+    end
+
+    # Replaces resources of the same type to the collection related to the
+    # '@belongs_to' resource. An empty array removes all resources.
+    def replace_relationships(items)
+      body = relationship_request_body(items)
+
+      @client.patch(relationship_path, body: body) if relationship_path
```

Comment:
can we get rid of the `if relationship_path`s now that it either returns something or raises an error?